cmd/tke-installer/app/installer/constants/constants.go

```diff
@@ -24,6 +24,7 @@ const (
 	ClusterLogFile = DataDir + "tke.log"
 
 	DefaultCustomResourceDir  = DataDir + "custom_upgrade_resource"
+	DefaultCustomChartsName   = "custom.charts.tar.gz"
```

Comment:
after think more, I aware that we don't need DefaultCustomChartsName  , just let user named the chart